scripts/bump.js

```diff
@@ -0,0 +1,25 @@
+/* global process, require */
+
+const fs = require( 'fs' );
+const shell = require( 'shelljs' );
+
+const args = process.argv;
+
+if ( !( args && args[ 2 ] && args[ 2 ].length > 2 ) ) {
+	console.log( 'Missing CKEditor version! USAGE: npm run bump -- A.B.C, for example: npm run bump -- 4.11.5' );
+} else {
+	const version = args[ 2 ];
+
+	// Update version in 'src/ckeditor.jsx' file.
+	const filePath = './src/ckeditor.jsx';
+	const fileData = fs.readFileSync( filePath, 'utf8' );
+
+	fs.writeFileSync( filePath,
+		fileData.replace( /https:\/\/cdn\.ckeditor\.com\/\d\.\d+\.\d+/g, `https://cdn.ckeditor.com/${ version }` ), 'utf8' );
+
+	// Update 'peerDependency' in 'package.json'. See `json` package docs - http://trentm.com/json.
+	shell.exec( `./node_modules/json/lib/json.js -I -f package.json -e 'this.peerDependencies.ckeditor="^${ version }"'` );
```

Comment:
`json` package is actually not needed as dependency. JSON files can be loaded in Node as objects using simply `require`. This way we can replace this dependency with updating and serializing required object:

```javascript
const pkg = require( '../package.json' );

pkg.peerDependencies.ckeditor = `^${ version }`;

fs.writeFileSync( path.resolve( __dirname, '..', 'package.json' ), JSON.stringify( pkg ) );
```

We can even control the formatting of saved JSON, using `JSON.stringify` parametes ;)